Foundation/NSOrderedSet.swift

```diff
@@ -243,25 +232,61 @@ extension NSOrderedSet {
     // to the original ordered set will "show through" the facade and it will
     // appear to change spontaneously, since a copy of the ordered set is not
     // being made.
-    public var array: [AnyObject] { NSUnimplemented() }
-    public var set: Set<NSObject> { NSUnimplemented() }
-    
-    public func enumerateObjectsUsingBlock(_ block: (AnyObject, Int, UnsafeMutablePointer<ObjCBool>) -> Void) { NSUnimplemented() }
-    public func enumerateObjectsWithOptions(_ opts: NSEnumerationOptions, usingBlock block: (AnyObject, Int, UnsafeMutablePointer<ObjCBool>) -> Void) { NSUnimplemented() }
-    public func enumerateObjectsAtIndexes(_ s: NSIndexSet, options opts: NSEnumerationOptions, usingBlock block: (AnyObject, Int, UnsafeMutablePointer<ObjCBool>) -> Void) { NSUnimplemented() }
+    public var array: [AnyObject] {
+        return _orderedStorage.bridge()
```

Comment:
Yeah. I didn't come up with any solution here. Thus return just a copy of the `array`/`set`. Will work on implementing other `NSOrderedSet` methods and think over about improving these ones then.